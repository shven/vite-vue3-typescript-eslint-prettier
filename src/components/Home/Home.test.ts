import { shallowMount } from '@vue/test-utils';
import Component from './VueHome.vue';

describe('VueIcon.vue', () => {
    const wrapper = shallowMount(Component);

    it('Match snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
