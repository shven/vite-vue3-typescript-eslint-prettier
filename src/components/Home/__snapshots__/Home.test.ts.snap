// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VueIcon.vue > Match snapshot 1`] = `
<div>
  <vue-container-stub />
</div>
`;
